### ISSUE TITLE
psm3 'Building src checksum...' step fails with out of tree builds

### ISSUE THREAD

**Issue by chuckcranor** [ofiwg/libfabric#6630]
If the psm3 provider is enabled and you do an out of source tree build, the compile will generate many 'cat' errors related to 'Building src checksum...'    The 'cat' errors do not abort the build, but clearly something is broken.

The errors go away if you configure with "--disable-psm3" ...

You can repeat this test with (assumes auto setting will enable psm3):
<pre>
git clone https://github.com/ofiwg/libfabric.git
( cd libfabric  && ./autogen.sh )
mkdir build
cd build
../libfabric/configure 
make
</pre>

Sample output (generated by rerunning "make").  Note the 'cat' errors (due to out of tree build) do not generate a make error, so the checksum it is generating is likely bogus.

<pre>
h0:final/build % make
make  all-am
make[1]: Entering directory '/tmp/final/build'
Building src checksum...
cat: prov/psm3/src/version.h: No such file or directory
cat: prov/psm3/src/psmx3.h: No such file or directory
cat: prov/psm3/src/psmx3_am.c: No such file or directory
cat: prov/psm3/src/psmx3_atomic.c: No such file or directory
cat: prov/psm3/src/psmx3_attr.c: No such file or directory
cat: prov/psm3/src/psmx3_av.c: No such file or directory
cat: prov/psm3/src/psmx3_cm.c: No such file or directory
cat: prov/psm3/src/psmx3_cntr.c: No such file or directory
cat: prov/psm3/src/psmx3_cq.c: No such file or directory
cat: prov/psm3/src/psmx3_domain.c: No such file or directory
cat: prov/psm3/src/psmx3_ep.c: No such file or directory
cat: prov/psm3/src/psmx3_fabric.c: No such file or directory
cat: prov/psm3/src/psmx3_init.c: No such file or directory
cat: prov/psm3/src/psmx3_mr.c: No such file or directory
cat: prov/psm3/src/psmx3_msg.c: No such file or directory
cat: prov/psm3/src/psmx3_rma.c: No such file or directory
cat: prov/psm3/src/psmx3_tagged.c: No such file or directory
cat: prov/psm3/src/psmx3_trigger.h: No such file or directory
cat: prov/psm3/src/psmx3_trx_ctxt.c: No such file or directory
cat: prov/psm3/src/psmx3_util.c: No such file or directory
cat: prov/psm3/src/psmx3_wait.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/am_config.h: No such file or directory
cat: prov/psm3/psm3/ptl_am/am_cuda_memhandle_cache.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/am_cuda_memhandle_cache.h: No such file or directory
cat: prov/psm3/psm3/ptl_am/am_reqrep.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/am_reqrep_shmem.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/cmarw.h: No such file or directory
cat: prov/psm3/psm3/ptl_am/cmarwu.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/psm_am_internal.h: No such file or directory
cat: prov/psm3/psm3/ptl_am/ptl.c: No such file or directory
cat: prov/psm3/psm3/ptl_am/ptl_fwd.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_config.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_crc32.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_epstate.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_epstate.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_expected_proto.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_opp_path_rec.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_path_rec.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_path_rec.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_am.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_am.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_connect.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_dump.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_expected.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_header.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_help.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_internal.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_mq.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_params.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_proto_recv.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_recvhdrq.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_recvhdrq.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_recvq.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_recvq.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_scb.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_scb.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_stats.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_subcontext.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tid.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tid.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tidcache.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tidcache.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tidflow.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_tidflow.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_writehdrq.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ips_writehdrq.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ptl.c: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ptl_fwd.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ptl_ips.h: No such file or directory
cat: prov/psm3/psm3/ptl_ips/ptl_rcvthread.c: No such file or directory
cat: prov/psm3/psm3/ptl_self/ptl.c: No such file or directory
cat: prov/psm3/psm3/ptl_self/ptl_fwd.h: No such file or directory
cat: prov/psm3/psm3/libuuid/pack.c: No such file or directory
cat: prov/psm3/psm3/libuuid/parse.c: No such file or directory
cat: prov/psm3/psm3/libuuid/psm_uuid.c: No such file or directory
cat: prov/psm3/psm3/libuuid/psm_uuid.h: No such file or directory
cat: prov/psm3/psm3/libuuid/unpack.c: No such file or directory
cat: prov/psm3/psm3/libuuid/unparse.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_debug.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_dwordcpy-x86_64.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_service.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_sysfs.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_syslog.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_time.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_utils.c: No such file or directory
cat: prov/psm3/psm3/include/opa_byteorder.h: No such file or directory
cat: prov/psm3/psm3/include/opa_debug.h: No such file or directory
cat: prov/psm3/psm3/include/opa_intf.h: No such file or directory
cat: prov/psm3/psm3/include/opa_queue.h: No such file or directory
cat: prov/psm3/psm3/include/opa_revision.h: No such file or directory
cat: prov/psm3/psm3/include/opa_service.h: No such file or directory
cat: prov/psm3/psm3/include/opa_udebug.h: No such file or directory
cat: prov/psm3/psm3/include/opa_user.h: No such file or directory
cat: prov/psm3/psm3/include/psm2_mock_testing.h: No such file or directory
cat: prov/psm3/psm3/include/rbtree.h: No such file or directory
cat: prov/psm3/psm3/include/linux-i386/bit_ops.h: No such file or directory
cat: prov/psm3/psm3/include/linux-i386/sysdep.h: No such file or directory
cat: prov/psm3/psm3/mpspawn/mpspawn_stats.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/hfi1_deprecated_gen1.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_common_gen1.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_i2cflash_gen1.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_proto_gen1.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_service_gen1.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_service_gen1.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_user_gen1.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/opa_utils_gen1.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_gdrcpy.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_hal_gen1.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_hal_gen1.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_hal_inline_i.h: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_hal_gen1_spio.h: No such file or directory
cat: prov/psm3/psm3/psm.c: No such file or directory
cat: prov/psm3/psm3/psm_am.c: No such file or directory
cat: prov/psm3/psm3/psm_am_internal.h: No such file or directory
cat: prov/psm3/psm3/psm_config.h: No such file or directory
cat: prov/psm3/psm3/psm_context.c: No such file or directory
cat: prov/psm3/psm3/psm_context.h: No such file or directory
cat: prov/psm3/psm3/psm_diags.c: No such file or directory
cat: prov/psm3/psm3/psm_ep.c: No such file or directory
cat: prov/psm3/psm3/psm_ep.h: No such file or directory
cat: prov/psm3/psm3/psm_ep_connect.c: No such file or directory
cat: prov/psm3/psm3/psm_error.c: No such file or directory
cat: prov/psm3/psm3/psm_error.h: No such file or directory
cat: prov/psm3/psm3/psm_gdrcpy.h: No such file or directory
cat: prov/psm3/psm3/psm_help.h: No such file or directory
cat: prov/psm3/psm3/psm_lock.h: No such file or directory
cat: prov/psm3/psm3/psm_log.h: No such file or directory
cat: prov/psm3/psm3/psm_memcpy.c: No such file or directory
cat: prov/psm3/psm3/psm_mock.c: No such file or directory
cat: prov/psm3/psm3/psm_mpool.c: No such file or directory
cat: prov/psm3/psm3/psm_mpool.h: No such file or directory
cat: prov/psm3/psm3/psm_mq.c: No such file or directory
cat: prov/psm3/psm3/psm_mq_internal.h: No such file or directory
cat: prov/psm3/psm3/psm_mq_recv.c: No such file or directory
cat: prov/psm3/psm3/psm_mq_utils.c: No such file or directory
cat: prov/psm3/psm3/psm_netutils.h: No such file or directory
cat: prov/psm3/psm3/psm_perf.c: No such file or directory
cat: prov/psm3/psm3/psm_perf.h: No such file or directory
cat: prov/psm3/psm3/psm_rndv_mod.c: No such file or directory
cat: prov/psm3/psm3/psm_rndv_mod.h: No such file or directory
cat: prov/psm3/psm3/psm_stats.c: No such file or directory
cat: prov/psm3/psm3/psm_stats.h: No such file or directory
cat: prov/psm3/psm3/psm_sysbuf.c: No such file or directory
cat: prov/psm3/psm3/psm_sysbuf.h: No such file or directory
cat: prov/psm3/psm3/psm_timer.c: No such file or directory
cat: prov/psm3/psm3/psm_timer.h: No such file or directory
cat: prov/psm3/psm3/psm_user.h: No such file or directory
cat: prov/psm3/psm3/psm_utils.c: No such file or directory
cat: prov/psm3/psm3/psm_utils.h: No such file or directory
cat: prov/psm3/psm3/psm_verbs_ep.c: No such file or directory
cat: prov/psm3/psm3/psm_verbs_ep.h: No such file or directory
cat: prov/psm3/psm3/psm_verbs_mr.c: No such file or directory
cat: prov/psm3/psm3/psm_verbs_mr.h: No such file or directory
cat: prov/psm3/psm3/psm_udp_ep.c: No such file or directory
cat: prov/psm3/psm3/psm_udp_ep.h: No such file or directory
cat: prov/psm3/psm3/psmi_wrappers.c: No such file or directory
cat: prov/psm3/psm3/psmi_wrappers.h: No such file or directory
cat: prov/psm3/psm3/psm2.h: No such file or directory
cat: prov/psm3/psm3/psm2_am.h: No such file or directory
cat: prov/psm3/psm3/psm2_hal.c: No such file or directory
cat: prov/psm3/psm3/psm2_hal.h: No such file or directory
cat: prov/psm3/psm3/psm2_hal_inline_t.h: No such file or directory
cat: prov/psm3/psm3/psm2_mq.h: No such file or directory
cat: prov/psm3/psm3/ptl.h: No such file or directory
cat: prov/psm3/psm3/include/rbtree.c: No such file or directory
cat: prov/psm3/psm3/psm_hal_gen1/psm_hal_gen1_spio.c: No such file or directory
cat: prov/psm3/psm3/opa/opa_dwordcpy-x86_64-fast.S: No such file or directory
cat: NEWS.md: No such file or directory
cat: libfabric.spec.in: No such file or directory
cat: config/distscript.pl: No such file or directory
cat: man/man1/fi_info.1: No such file or directory
cat: man/man1/fi_pingpong.1: No such file or directory
cat: man/man1/fi_strerror.1: No such file or directory
cat: man/man3/fi_av.3: No such file or directory
cat: man/man3/fi_av_set.3: No such file or directory
cat: man/man3/fi_cm.3: No such file or directory
cat: man/man3/fi_cntr.3: No such file or directory
cat: man/man3/fi_collective.3: No such file or directory
cat: man/man3/fi_control.3: No such file or directory
cat: man/man3/fi_cq.3: No such file or directory
cat: man/man3/fi_domain.3: No such file or directory
cat: man/man3/fi_endpoint.3: No such file or directory
cat: man/man3/fi_errno.3: No such file or directory
cat: man/man3/fi_eq.3: No such file or directory
cat: man/man3/fi_fabric.3: No such file or directory
cat: man/man3/fi_getinfo.3: No such file or directory
cat: man/man3/fi_mr.3: No such file or directory
cat: man/man3/fi_msg.3: No such file or directory
cat: man/man3/fi_nic.3: No such file or directory
cat: man/man3/fi_poll.3: No such file or directory
cat: man/man3/fi_rma.3: No such file or directory
cat: man/man3/fi_tagged.3: No such file or directory
cat: man/man3/fi_trigger.3: No such file or directory
cat: man/man3/fi_version.3: No such file or directory
cat: man/man7/fabric.7: No such file or directory
cat: man/man7/fi_hook.7: No such file or directory
cat: man/man7/fi_provider.7: No such file or directory
cat: man/man7/fi_direct.7: No such file or directory
cat: man/man7/fi_sockets.7: No such file or directory
cat: man/man7/fi_udp.7: No such file or directory
cat: man/man7/fi_verbs.7: No such file or directory
cat: man/man7/fi_efa.7: No such file or directory
cat: man/man7/fi_usnic.7: No such file or directory
cat: man/man7/fi_psm.7: No such file or directory
cat: man/man7/fi_psm2.7: No such file or directory
cat: man/man7/fi_psm3.7: No such file or directory
cat: man/man7/fi_gni.7: No such file or directory
cat: man/man7/fi_rxm.7: No such file or directory
cat: man/man7/fi_mrail.7: No such file or directory
cat: man/man7/fi_bgq.7: No such file or directory
cat: man/man7/fi_shm.7: No such file or directory
cat: man/man7/fi_tcp.7: No such file or directory
cat: man/man7/fi_rstream.7: No such file or directory
cat: man/man3/fi_accept.3: No such file or directory
cat: man/man3/fi_alias.3: No such file or directory
cat: man/man3/fi_allgather.3: No such file or directory
cat: man/man3/fi_allreduce.3: No such file or directory
cat: man/man3/fi_alltoall.3: No such file or directory
cat: man/man3/fi_atomic.3: No such file or directory
cat: man/man3/fi_atomic_valid.3: No such file or directory
cat: man/man3/fi_atomicmsg.3: No such file or directory
cat: man/man3/fi_atomicv.3: No such file or directory
cat: man/man3/fi_av_bind.3: No such file or directory
cat: man/man3/fi_av_insert.3: No such file or directory
cat: man/man3/fi_av_insertsvc.3: No such file or directory
cat: man/man3/fi_av_lookup.3: No such file or directory
cat: man/man3/fi_av_open.3: No such file or directory
cat: man/man3/fi_av_remove.3: No such file or directory
cat: man/man3/fi_av_set_diff.3: No such file or directory
cat: man/man3/fi_av_set_insert.3: No such file or directory
cat: man/man3/fi_av_set_intersect.3: No such file or directory
cat: man/man3/fi_av_set_remove.3: No such file or directory
cat: man/man3/fi_av_set_union.3: No such file or directory
cat: man/man3/fi_av_straddr.3: No such file or directory
cat: man/man3/fi_barrier.3: No such file or directory
cat: man/man3/fi_broadcast.3: No such file or directory
cat: man/man3/fi_cancel.3: No such file or directory
cat: man/man3/fi_close.3: No such file or directory
cat: man/man3/fi_cntr_add.3: No such file or directory
cat: man/man3/fi_cntr_open.3: No such file or directory
cat: man/man3/fi_cntr_read.3: No such file or directory
cat: man/man3/fi_cntr_set.3: No such file or directory
cat: man/man3/fi_cntr_wait.3: No such file or directory
cat: man/man3/fi_compare_atomic.3: No such file or directory
cat: man/man3/fi_compare_atomic_valid.3: No such file or directory
cat: man/man3/fi_compare_atomicmsg.3: No such file or directory
cat: man/man3/fi_compare_atomicv.3: No such file or directory
cat: man/man3/fi_connect.3: No such file or directory
cat: man/man3/fi_cq_open.3: No such file or directory
cat: man/man3/fi_cq_read.3: No such file or directory
cat: man/man3/fi_cq_readerr.3: No such file or directory
cat: man/man3/fi_cq_readfrom.3: No such file or directory
cat: man/man3/fi_cq_sread.3: No such file or directory
cat: man/man3/fi_cq_sreadfrom.3: No such file or directory
cat: man/man3/fi_cq_strerror.3: No such file or directory
cat: man/man3/fi_cq_signal.3: No such file or directory
cat: man/man3/fi_domain_bind.3: No such file or directory
cat: man/man3/fi_domain_query.3: No such file or directory
cat: man/man3/fi_dupinfo.3: No such file or directory
cat: man/man3/fi_enable.3: No such file or directory
cat: man/man3/fi_ep.3: No such file or directory
cat: man/man3/fi_ep_bind.3: No such file or directory
cat: man/man3/fi_eq_open.3: No such file or directory
cat: man/man3/fi_eq_read.3: No such file or directory
cat: man/man3/fi_eq_readerr.3: No such file or directory
cat: man/man3/fi_eq_sread.3: No such file or directory
cat: man/man3/fi_eq_strerror.3: No such file or directory
cat: man/man3/fi_eq_write.3: No such file or directory
cat: man/man3/fi_fetch_atomic.3: No such file or directory
cat: man/man3/fi_fetch_atomic_valid.3: No such file or directory
cat: man/man3/fi_fetch_atomicmsg.3: No such file or directory
cat: man/man3/fi_fetch_atomicv.3: No such file or directory
cat: man/man3/fi_freeinfo.3: No such file or directory
cat: man/man3/fi_getname.3: No such file or directory
cat: man/man3/fi_getopt.3: No such file or directory
cat: man/man3/fi_getpeer.3: No such file or directory
cat: man/man3/fi_inject.3: No such file or directory
cat: man/man3/fi_injectdata.3: No such file or directory
cat: man/man3/fi_inject_atomic.3: No such file or directory
cat: man/man3/fi_inject_write.3: No such file or directory
cat: man/man3/fi_inject_writedata.3: No such file or directory
cat: man/man3/fi_join.3: No such file or directory
cat: man/man3/fi_join_collective.3: No such file or directory
cat: man/man3/fi_leave.3: No such file or directory
cat: man/man3/fi_listen.3: No such file or directory
cat: man/man3/fi_mr_bind.3: No such file or directory
cat: man/man3/fi_mr_desc.3: No such file or directory
cat: man/man3/fi_mr_key.3: No such file or directory
cat: man/man3/fi_mr_reg.3: No such file or directory
cat: man/man3/fi_mr_regattr.3: No such file or directory
cat: man/man3/fi_mr_regv.3: No such file or directory
cat: man/man3/fi_open.3: No such file or directory
cat: man/man3/fi_open_ops.3: No such file or directory
cat: man/man3/fi_passive_ep.3: No such file or directory
cat: man/man3/fi_pep_bind.3: No such file or directory
cat: man/man3/fi_poll_add.3: No such file or directory
cat: man/man3/fi_poll_del.3: No such file or directory
cat: man/man3/fi_poll_open.3: No such file or directory
cat: man/man3/fi_query_atomic.3: No such file or directory
cat: man/man3/fi_query_collective.3: No such file or directory
cat: man/man3/fi_read.3: No such file or directory
cat: man/man3/fi_readmsg.3: No such file or directory
cat: man/man3/fi_readv.3: No such file or directory
cat: man/man3/fi_recv.3: No such file or directory
cat: man/man3/fi_recvmsg.3: No such file or directory
cat: man/man3/fi_recvv.3: No such file or directory
cat: man/man3/fi_reduce_scatter.3: No such file or directory
cat: man/man3/fi_reject.3: No such file or directory
cat: man/man3/fi_rx_addr.3: No such file or directory
cat: man/man3/fi_rx_size_left.3: No such file or directory
cat: man/man3/fi_scalable_ep_bind.3: No such file or directory
cat: man/man3/fi_send.3: No such file or directory
cat: man/man3/fi_senddata.3: No such file or directory
cat: man/man3/fi_sendmsg.3: No such file or directory
cat: man/man3/fi_sendv.3: No such file or directory
cat: man/man3/fi_setname.3: No such file or directory
cat: man/man3/fi_setopt.3: No such file or directory
cat: man/man3/fi_shutdown.3: No such file or directory
cat: man/man3/fi_strerror.3: No such file or directory
cat: man/man3/fi_tinject.3: No such file or directory
cat: man/man3/fi_tinjectdata.3: No such file or directory
cat: man/man3/fi_tostr.3: No such file or directory
cat: man/man3/fi_trecv.3: No such file or directory
cat: man/man3/fi_trecvmsg.3: No such file or directory
cat: man/man3/fi_trecvv.3: No such file or directory
cat: man/man3/fi_trywait.3: No such file or directory
cat: man/man3/fi_tsend.3: No such file or directory
cat: man/man3/fi_tsenddata.3: No such file or directory
cat: man/man3/fi_tsendmsg.3: No such file or directory
cat: man/man3/fi_tsendv.3: No such file or directory
cat: man/man3/fi_tx_size_left.3: No such file or directory
cat: man/man3/fi_wait.3: No such file or directory
cat: man/man3/fi_wait_open.3: No such file or directory
cat: man/man3/fi_write.3: No such file or directory
cat: man/man3/fi_writedata.3: No such file or directory
cat: man/man3/fi_writemsg.3: No such file or directory
cat: man/man3/fi_writev.3: No such file or directory
SRC checksum not changed: da39a3ee5e6b4b0d3255bfef95601890afd80709
Updated build timestamp: Fri Mar 12 15:32:10 EST 2021
make[1]: Leaving directory '/tmp/final/build'
h0:final/build % 
</pre>

**Comment by acgoldma** [comment]
There are 2 issues here.

1. The man pages should not be in the checksum. That was caused by using `EXTA_DIST` in chksum files. Need to move that do just include the extra files in psm3.
2. Need to prefix each file with `$(top_srcdir)`, will have to change the command to something like:
```
chksum=`for file in $(chksum_srcs); do cat $(top_srcdir)/$$file; done | sha1sum | cut -d' ' -f 1`; \
```

**Comment by acgoldma** [comment]
I was able to reproduce on my system, and was able to fix it with #6631 
This patch also removes the manpages and only checksums the psm3 source files.
